### PR TITLE
Tag Azure resource group and namespace with the Kyma runtime properties

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Tag Azure resource group and namespace with the Kyma runtime properties:

- SubAccountID
- InstanceID
- OperationID

**Related issue(s)**
- https://github.com/kyma-incubator/compass/issues/967.

### CODE REVIEW

File: components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go [hunk 1]
```diff
@@ -51,13 +51,14 @@ func (nc *FakeNamespaceClient) GetEventhubAccessKeys(ctx context.Context, resour
 	}, nc.accessKeysError
 }
 
-func (nc *FakeNamespaceClient) CreateResourceGroup(ctx context.Context, config *azure.Config, name string) (resources.Group, error) {
+func (nc *FakeNamespaceClient) CreateResourceGroup(ctx context.Context, config *azure.Config, name string, tags azure.Tags) (resources.Group, error) {
```
Comment (nachtmaar) [inline components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go:54]: Can we test that the tags are passed in a unit test? 🤔

File: components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go [hunk 2]
```diff
@@ -91,10 +89,16 @@ func (p *ProvisionAzureEventHubStep) Run(operation internal.ProvisioningOperatio
 		return p.operationManager.OperationFailed(operation, errorMessage)
 	}
 
+	// prepare azure tags
+	tags := make(azure.Tags, 3)
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go]: Done.

File: components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go (hunk 1 above)
Comment (marcobebway) [reply components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go:61]: Done.

File: components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go [hunk 3]
```diff
@@ -415,20 +426,22 @@ func fixAccountProviderGardenerCredentialsError() hyperscalerautomock.AccountPro
...
 	op := internal.ProvisioningOperation{
-		Operation: internal.Operation{},
+		Operation: internal.Operation{
+			ID:         fixOperationID,
+			InstanceID: fixInstanceID,
+		},
 		ProvisioningParameters: `{
 			"plan_id": "4deee563-e5ec-4731-b9b1-53b42d855f0c",
+			"ers_context": {
+        		"subaccount_id": "` + fixSubAccountID + `"
```
Comment (nachtmaar) [inline components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go:443]: (nit) indention

File: components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go [hunk 4]
```diff
@@ -415,20 +426,22 @@ func fixAccountProviderGardenerCredentialsError() hyperscalerautomock.AccountPro
...
 func fixProvisioningOperation(t *testing.T) internal.ProvisioningOperation {
 	op := internal.ProvisioningOperation{
-		Operation: internal.Operation{},
+		Operation: internal.Operation{
+			ID:         fixOperationID,
+			InstanceID: fixInstanceID,
+		},
 		ProvisioningParameters: `{
 			"plan_id": "4deee563-e5ec-4731-b9b1-53b42d855f0c",
+			"ers_context": {
```
Comment (nachtmaar) [inline components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go:442]: Shouldn't it be "context" ?

File: components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go (hunk 3 above)
Comment (marcobebway) [reply components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go:443]: Github is not showing this correctly.

Comment (nachtmaar) [reply components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go:443]: problem is mixture of spaces and tabs ;)

Comment (marcobebway) [reply components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go]: Rewrote it all.

**Comment by jasiu001** [comment]
lgtm

**Pull request merged**